src/commands/synthetics/utils.ts

```diff
@@ -844,8 +881,6 @@ export const renderResults = ({
     } else {
       summary.failed++
     }
-
-    reporter.resultEnd(result, getAppBaseURL(config))
```

Comment:
Since we no longer print the results as part of this function, could we rename the function from `renderResults` to something like `renderResultSummary`?

Reply:
This is a breaking change and I wanted to avoid that. But I can deprecate the old one with:

```ts
/**
 * @deprecated Use filterAndFormatGitRemote instead
 */
export const renderResults = renderResultSummary
```

Reply:
👍 let's do that.

Reply:
Per conversation, we'll address this in a follow-up